maestro-client/src/main/java/maestro/Errors.kt

```diff
@@ -19,6 +19,10 @@
 
 package maestro
 
+data class DebugInfo (
+    val debugMessage: String
```

Comment:
what do we expect this object to contain, other than a message?

Reply:
Yeah I have changed it back to string for now, from the discussion yesterday. Want to keep it simple.